utils/vlog/tests/fig28-lutff-pair/lut/golden.pb_type.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version='1.0' encoding='utf-8'?>
+<pb_type xmlns:xi="http://www.w3.org/2001/XInclude" blif_model=".subckt lut4" name="LUT4" num_pb="1">
```

Comment:
Shouldn't this be CLASS = LUT, rather than subckt = lut4?

Reply:
@kgugala